AGENTS.md

```
@@ -18,4 +18,185 @@
+| `Homepage Tooltip Colour`         | Tooltip text colour                        | `#424242`     |
+| `Homepage Section Header Colour`  | Colour for section headers                 | `#263238`     |
+| `Critical Alert Colour`           | Colour for critical messages               | `#D32F2F`     |
+| `Warning Alert Colour`            | Colour for warnings                        | `#FBC02D`     |
+| `Success Highlight Colour`        | Colour for success states                  | `#388E3C`     |
+
+### Sample Access in JSF
+```xhtml
+style="color: #{configOptionApplicationController.getShortTextValueByKey('Home Icon Colour', true)}"
+```
+
+## Homepage Icon Management Guidelines
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content
null

(MD024, no-duplicate-heading)